Tiltfile

```diff
@@ -283,6 +283,7 @@ def deploy_worker_templates(template, substitutions):
         yaml = yaml.replace("${" + substitution + "}", value)
 
     yaml = yaml.replace('"', '\\"')     # add escape character to double quotes in yaml
+    yaml = yaml.replace('$', '\\$')     # add escape character to dollar sign chars in yaml
```

Comment:
If we're not doing tilt integration in this PR let's keep this change out for now and add it when the template can actually be used with tilt

Reply:
same with the tilt specific docs that say it can be used with the UI